build.gradle

```diff
@@ -17,6 +17,9 @@ buildscript {
     }
 }
 
+apply plugin: 'application'
+mainClassName='main.Main'
```

Comment:
This plugin didn't do the job I wanted. We need to ask Gradle to modify the MANIFEST.MF file in the jar that it generates. This plugin seems to be giving us shell scripts to configure the program, and indirectly running the .jar through the shell scripts. If I'm right, we'll be using launch4j for this purpose, making the shell scripts kind of redundant.

Use the manifest plugin to write the main class to the MANIFEST.MF file. This will make the jar executable through ```java -jar BakeoutController.jar```.